src/css/normalize.css

```diff
@@ -200,7 +200,7 @@ html {
 }
 body {
   color: rgba(0, 0, 0, 0.8);
-  font-family: ff-clan-web-pro, '-apple-system', BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen,
+  font-family: '-apple-system', BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen,
```

Comment:
No quotes needed for the `-apple-system` and `sans-serif` I guess

Reply:
Yeah saw your update and fixed :)